Source/Analytics/Web/src/components/lastWeekTotals/LastWeekTotals.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from "react";
+
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Typography from '@material-ui/core/Typography';
+import Paper from '@material-ui/core/Paper';
+
+import { CaseReportTotalsQuery } from "../../Features/Overview/LastWeekTotals/CaseReportTotalsQuery";
+import { QueryCoordinator } from "@dolittle/queries";
+
+import './last-week-totals.scss';
+
+export default class LastWeekTotals extends Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            femalesUnder5: "-",
+            malesUnder5: "-",
+            femalesOver5: "-",
+            malesOver5: "-",
+            isLoading: true,
+            isError: false
+        };
+    }
+
+    fetchLastWeekTotals() {
+        this.queryCoordinator = new QueryCoordinator();
+        let caseReportTotals = new CaseReportTotalsQuery();
+
+        this.queryCoordinator.execute(caseReportTotals).then(queryResult => {
+            if(queryResult.success){
+                let json = queryResult.items[0];
+                this.setState({
+                    femalesUnder5: json.femalesUnder5,
+                    malesUnder5: json.malesUnder5,
+                    femalesOver5: json.femalesOver5,
+                    malesOver5: json.malesOver5,
+                    isLoading: false,
+                    isError: false
+                });
+            }
+        }).catch(_ => {
+            this.setState({
+                isLoading: false,
+                isError: true
+            })
+        }
+        )
+    }
+
+    componentDidMount() {
+        this.fetchLastWeekTotals();
+    }
+
+    render () {
+        var totalUnder5 = parseInt(this.state.femalesUnder5) + parseInt(this.state.malesUnder5);
```

Comment:
Is it better to parse the numbers in fetchLastWeekTotals?

Reply:
Yes! I was worried this would cause a problem as the state starts out as a string `"-"`, but then I remembered that this is JS so it's okay 🐒